config/config.go

```diff
@@ -51,6 +51,15 @@ const (
 
 	SettingDbUsername = "mongo_username"
 	SettingDbPassword = "mongo_password"
+
+	SettingLimitTokensPerUser        = "limit_tokens_per_user"
+	SettingLimitTokensPerUserDefault = 10
+
+	SettingTokenLastUsedUpdateFreqMinutes        = "token_last_used_update_freq_minutes"
+	SettingTokenLastUsedUpdateFreqMinutesDefault = 5
+
+	SettingTokenMaxExpirationSeconds        = "token_max_expiration_seconds"
+	SettingTokenMaxExpirationSecondsDefault = 31536000
```

Comment:
this is a year, just please confirm, is that what we want?

Reply:
yes, it's ~1 year and this is what we want; still, it's configurable, so if we'll change our mind it will be easy to provide different value